### PR TITLE
Adding DRTs

### PR DESCRIPTION
**Pull request by rchauhan18**
Adding DRTS

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/wpf-test?pullRequest=27) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/wpf-test?pullRequest=27) before we can accept your contribution.<br/><br/>:x: rchauhan18 [sign now](https://cla.dotnetfoundation.org/dotnet/wpf-test?pullRequest=27)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/wpf-test?pullRequest=27) it.</sub>

**Review by Kuldeep-MS — APPROVED** [review]
Please address the comment given. Overall looks good to me.

**Review by dipeshmsft — APPROVED** [review]

File: src/Directory.Build.props [hunk 1]
```diff
@@ -8,7 +8,19 @@
...
+    <!-- SYSLIB0018: 'Assembly.ReflectionOnlyLoadFrom(string)' is obsolete -->
+
+    <NoWarn>$(NoWarn);SYSLIB0021;SYSLIB0024;SYSLIB0018</NoWarn>
+    <!-- Temporarily disable more warnings (10/28/2021) -->
+    <!-- SYSLIB0032: ''HandleProcessCorruptedStateExceptionsAttribute' is obsolete -->
+    <!-- CA2018:  'Buffer.BlockCopy' expects the number of bytes to be copied for the 'count' argument -->
+    <NoWarn>$(NoWarn);CA2018;SYSLIB0032;</NoWarn>
   </PropertyGroup>
-</Project>
+</Project>
```
Comment (pchaurasia14) [inline src/Directory.Build.props:26]: @rchauhan18  - Add a new line to get rid of the warning shown here

File: src/Test/Annotations/Common/Library/AnnotationCommon.asmmeta.config.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<root>
+    <assembly publish="$(WPFINTRA_REF_PATH)\AnnotationCommon.dll" preserveInternals="0"/>
+</root>
```
Comment (pchaurasia14) [inline src/Test/Annotations/Common/Library/AnnotationCommon.asmmeta.config.xml:4]: @rchauhan18  - Please add a newline here too

**Pull request merged**